preset/preset_cli_default.list

```diff
@@ -0,0 +1,9 @@
+<resources>
+    <section name="PresetTemplate">
+        <integer name="VersionMajor" value="15" />
+        <integer name="VersionMinor" value="0" />
+        <integer name="VersionMicro" value="0" />
+        <json name="Preset" file="preset_template.json" />
+    </section>
+    <json name="PresetCLIDefault" file="preset_cli_default.json" />
```

Comment:
Rather than creating a new preset_cli_default.h, I would just add the line
<json name="PresetCLIDefault" file="preset_cli_default.json" />
to the existing preset_builtin.h.  This can be loaded similar to how PresetBuiltin is loaded from the same hard coded string defined in preset_builtin.h.